### PR TITLE
[FIX] mail: chatter is disabled wasn't correctly computed

### PR DESCRIPTION
**Pull request by xdu-odoo**
task-2263933

### CODE REVIEW

**Review by seb-odoo — COMMENTED** [review]
Fix seems obviously good, but how did we notice it and can we cover it with a test?

**Comment by xdu-odoo** [comment]
You need to go once in create mode of a form view then open the form view of a record. This can be done too by creating a record and check directly the view after creation. I'm not sure if this is doable via a test ?

**Comment by xdu-odoo** [comment]
> You need to go once in create mode of a form view then open the form view of a record. This can be done too by creating a record and check directly the view after creation. I'm not sure if this is doable via a test ?

You little piece of shit, off course it's doable, look I did it !

File: addons/mail/static/src/messaging/entity/chatter/chatter.js [hunk 1]
```diff
@@ -106,10 +106,7 @@ function ChatterFactory({ Entity }) {
          * @returns {boolean}
          */
         _computeIsDisabled() {
-            if (!this.threadId) {
-                return true;
-            }
-            return this.isDisabled;
+            return !this.threadId;
```
Comment (seb-odoo) [inline addons/mail/static/src/messaging/entity/chatter/chatter.js:109]: Actually, it feels strange to have `threadId` stored in the composer instead of just `thread`, but this is out of scope.

**Review by seb-odoo — APPROVED** [review]
So I guess ok for me, @alexkuhn ?

File: addons/mail/static/src/messaging/widget/form_renderer/form_renderer_tests.js [hunk 2]
```diff
@@ -373,6 +373,81 @@ QUnit.test('chatter updating', async function (assert) {
...
+                    <field name="name"/>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_ids"/>
+                </div>
+            </form>
+        `,
+        viewOptions: {
+            mode: 'edit',
+        }
```
Comment (seb-odoo) [inline addons/mail/static/src/messaging/widget/form_renderer/form_renderer_tests.js]:
```suggestion
        },
```

**Comment by xdu-odoo** [comment]
Waiting for @alexkuhn final word :)

**Review by alexkuhn — APPROVED** [review]

**Comment by seb-odoo** [comment]
Ok squash and merge then
